i18n-config.js

```diff
@@ -85,6 +85,11 @@ module.exports = {
         short_name: 'BN',
         affiliate_lang: 15,
     },
+    ar: {
+        path: 'ar',
+        display_name: 'العربية',
+        short_name: 'العربية',
```

Comment:
Can we change this short name to 'AR' probably?

Reply:
done :+1: